From d285744e02c1d62475983174f35defec3f44ad16 Mon Sep 17 00:00:00 2001
From: Shunqing Chen <csq@rock-chips.com>
Date: Fri, 8 Dec 2017 17:53:36 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368-p9: add HDMI_SWITCH regulator

if not add HDMI_SWITCH node, h_5V will be disabled when resume

Change-Id: I06349a4e1f8c6caabaaa93f95db92ae514957d48
Signed-off-by: Shunqing Chen <csq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-p9.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
index 3bbd14c3a3b3..272e038a49ce 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
@@ -530,6 +530,15 @@
 			otg_switch: OTG_SWITCH {
 				regulator-name = "otg_switch";
 			};
+
+			hdmi_switch: HDMI_SWITCH {
+				regulator-always-on;
+				regulator-boot-on;
+				regulator-name = "hdmi_switch";
+				regulator-state-mem {
+					regulator-off-in-suspend;
+				};
+			};
 		};
 
 		battery {
-- 
2.35.3

